modules/core/shared/src/main/scala/lucuma/core/model/sequence/f2/F2StaticConfig.scala

```diff
@@ -5,29 +5,23 @@ package lucuma.core.model.sequence.f2
 
 import cats.Eq
 import cats.syntax.all.*
-import lucuma.core.enums.*
+import lucuma.core.enums.MosPreImaging
 import monocle.Focus
 import monocle.Lens
 
 case class F2StaticConfig(
-  mosPreImaging:          Boolean,
-  useElectronicOffseting: Boolean,
-  customSlitWidth:        Option[F2CustomSlitWidth]
+  mosPreImaging:          MosPreImaging,
```

Comment:
I learned this is only used only on imaging

Reply:
That's right, I think it is used to design a mask for MOS.